pkgs/development/python-modules/sphinxcontrib-katex/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, sphinx }:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-katex";
+  version = "0.5.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0qssq2qc12rnwb6yhw3xj69cwij3jp3sqzwb6n85qp36g4vxrkb6";
+  };
+
+  propagatedBuildInputs = [ sphinx ];
+
+  # There are no unit tests
+  doCheck = false;
```

Comment:
```suggestion
  doCheck = false;
  pythonImportsCheck = [ "sphinxcontrib.katex" ];
```

Reply:
Thanks! This check showed that the importing fails on Python 2, so I disabled the package for Python 2. SImilarly as for the other sphinxcontrib-fulltoc PR.